apps/lib/opt.c

```diff
@@ -149,7 +149,7 @@ char *opt_appname(const char *argv0)
     size_t len = strlen(prog);
 
     if (argv0 != NULL)
-        BIO_snprintf(prog + len, sizeof(prog) - len - 1, " %s", argv0);
+        snprintf(prog + len, sizeof(prog) - len - 1, " %s", argv0);
```

Comment:
There's a few of these. Looks like poorly reviewed conversion from what was previously strncpy.